.idea/goodwix-doctrine-json-odm.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please, remove `.idea` files from the pull request (the best way is to use the global `.gitignore` file).